roles/lsnr-create/tasks/main.yml

```diff
@@ -19,6 +19,8 @@
     permanent: yes
     immediate: yes
     state: enabled
+  ignore_errors: true
```

Comment:
ANy background on why we'd want to ignore errors?  Perhaps put as a comment in the file?

Reply:
if firewall is not running, the task fails saying it should be enabled. with offline: yes attribute (https://docs.ansible.com/ansible/latest/modules/firewalld_module.html) it was the same, so I did not find a way to ignore the task if firewall service is not running, so put ignore errors

Reply:
How about using failed_when (fail when we get a response code, and we don't see the string that might identify if firewalld isn't running)?

https://docs.ansible.com/ansible/latest/user_guide/playbooks_error_handling.html

Reply:
ok, put several conditions to avoid cases when firewall is not enabled and when firewall is already configured for the port